src/components/NewCommentForm/NewCommentForm.tsx

```diff
@@ -1,39 +1,105 @@
-import React from 'react';
 import './NewCommentForm.scss';
+import React, { FormEvent, useState } from 'react';
+import { Comment } from '../../react-app-env';
```

Comment:
Better to organize your imports like this:
```suggestion
import React, { FormEvent, useState } from 'react';
import { Comment } from '../../react-app-env';
import './NewCommentForm.scss';
```